{"name": "Проверка валидности поля номера телефона", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.html", "trace": "venv\\lib\\site-packages\\selene\\core\\wait.py:102: in for_\n    return fn(self._entity)\nvenv\\lib\\site-packages\\selene\\core\\condition.py:136: in __call__\n    return self._fn(*args, **kwargs)\nvenv\\lib\\site-packages\\selene\\core\\condition.py:91: in fn\n    if not predicate(entity):\nvenv\\lib\\site-packages\\selene\\core\\match.py:37: in <lambda>\n    'is visible', lambda element: element().is_displayed()\nvenv\\lib\\site-packages\\selene\\core\\entity.py:179: in __call__\n    return self._locator()\nvenv\\lib\\site-packages\\selene\\core\\locator.py:34: in __call__\n    return self._locate()\nvenv\\lib\\site-packages\\selene\\core\\entity.py:1529: in <lambda>\n    f'{self}.element({by})', lambda: self.driver.find_element(*by)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:976: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:321: in execute\n    self.error_handler.check_response(response)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\nE     (Session info: chrome=113.0.5672.94)\n\nDuring handling of the above exception, another exception occurred:\ntests\\test_login.py:90: in test_field_phone\n    page.registration_field_phone_number()\npages\\home.py:182: in registration_field_phone_number\n    self.wait_element(self.message_error_phone_text_var_string)\npages\\base.py:57: in wait_element\n    browser.s(locator).should(be.visible)\nvenv\\lib\\site-packages\\selene\\core\\entity.py:540: in should\n    super().should(condition)\nvenv\\lib\\site-packages\\selene\\core\\entity.py:127: in should\n    self.wait.for_(condition)\nvenv\\lib\\site-packages\\selene\\core\\wait.py:125: in for_\n    raise self._hook_failure(failure)\nE   selene.core.exceptions.TimeoutException: Message: \nE   \nE   Timed out after 10s, while waiting for:\nE   browser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\nE   \nE   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\nE     (Session info: chrome=113.0.5672.94)\nE   \nE   Screenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.png\nE   PageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.html"}, "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1684321275650, "stop": 1684321275697}, {"name": "Нажать Зарегистрироваться", "status": "passed", "start": 1684321275698, "stop": 1684321275849}, {"name": "Валидация поля Номер телефона", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\home.py\", line 182, in registration_field_phone_number\n    self.wait_element(self.message_error_phone_text_var_string)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 57, in wait_element\n    browser.s(locator).should(be.visible)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 540, in should\n    super().should(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 127, in should\n    self.wait.for_(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "steps": [{"name": "Заполнение поля '' текстом '999888776'", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616C1F0>"}, {"name": "text", "value": "999888776"}, {"name": "fieldName", "value": "''"}], "start": 1684321275849, "stop": 1684321275955}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616CF70>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1684321283019, "stop": 1684321283039}, {"name": "Заполнение поля '' текстом '999888775'", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616C1F0>"}, {"name": "text", "value": "999888775"}, {"name": "fieldName", "value": "''"}], "start": 1684321285041, "stop": 1684321285130}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616CF70>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1684321292175, "stop": 1684321292192}, {"name": "Заполнение поля '' текстом ''9998887m''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616C1F0>"}, {"name": "text", "value": "'9998887m'"}, {"name": "fieldName", "value": "''"}], "start": 1684321294204, "stop": 1684321294277}, {"name": "Взять текст из элемента ''Текст ошибки поля  пароль не верный или текст отсутствует''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616CF70>"}, {"name": "allureText", "value": "'Текст ошибки поля  пароль не верный или текст отсутствует'"}], "start": 1684321301320, "stop": 1684321301338}, {"name": "Заполнение поля '' текстом '9998887766'", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616C1F0>"}, {"name": "text", "value": "9998887766"}, {"name": "fieldName", "value": "''"}], "start": 1684321303347, "stop": 1684321303436}, {"name": "Проверка наличия элемента", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'некорректный номер телефона')]\")).is visible\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'некорректный номер телефона')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964495.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 57, in wait_element\n    browser.s(locator).should(be.visible)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 540, in should\n    super().should(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 127, in should\n    self.wait.for_(condition)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "parameters": [{"name": "locator", "value": "'//div[contains(text(),'некорректный номер телефона')]'"}], "start": 1684321310488, "stop": 1684321321001}], "start": 1684321275849, "stop": 1684321321002}], "attachments": [{"name": "Скринтшот", "source": "2fb823a4-8cbc-4e0d-bf4b-0ca0945a2474-attachment.png", "type": "image/png"}, {"name": "Логи консоли", "source": "675f6b7a-2d44-4b6c-b564-d9b79d657ff3-attachment.json", "type": "application/json"}, {"name": "log", "source": "79d3816e-d7b9-4dd8-a2f4-d5d2797e13b3-attachment.txt", "type": "text/plain"}], "start": 1684321275649, "stop": 1684321321002, "uuid": "c95377b3-0439-416a-b85f-2486b8b29544", "historyId": "98185df0bbce6dad3aecda5b36f83fa6", "testCaseId": "bbd874f4602a7fdc5ab9eaea7165381d", "fullName": "tests.test_login.TestLogin#test_field_phone", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "8372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}